ReduxSimple/ReduxStore.cs

```diff
@@ -46,5 +43,20 @@ public IObservable<object> ObserveAction()
         {
             return _actionSubject.OfType<T>().AsObservable();
         }
+
+        private void GoToState(TState state, object action)
```

Comment:
Not sure we need to extract the method. Based on my comments, the code should normally go back to `Dispatch` method.